Makefile

```
@@ -202,3 +202,7 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+.PHONY: rules/rules.go
+rules/rules.go: $(OAPI_CODEGEN)
```

From above's comment:
```suggestion
rules/rules.go: $(OAPI_CODEGEN) rules/spec.yaml
```